src/screenshots/covidcast/map_preview.py

```diff
@@ -0,0 +1,127 @@
+"""Takes a screenshot of the COVIDcast map.
+
+Extraneous UI is hidden so that the map is presentable in standalone form. The
+screenshot is intended to be suitable for use in social media previews and
+custom dashboards.
+"""
+
+# standard library
+import argparse
+import time
+import urllib.parse
+
+# third party
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium import webdriver
+
+# first party
+from delphi.operations.screenshots.utils import Utils
+
+
+class MapPreview:
+
+  # URL of the COVIDcast map with predetermined signals
+  URL = 'https://covidcast.cmu.edu/?' + urllib.parse.urlencode({
+    'sensor': 'doctor-visits-smoothed_adj_cli',
+    'level': 'county',
+    'date': '20200918',
```

Comment:
it would be great to specify some of these parameters as arguments, especially: 
 * date ... or empty such that it will use the latest
 * sensor
 * level

Furthermore, I you could remove some default values: signalType, mode, encoding

Reply:
Done: added all of these (and more) as args, and removed the ones you listed.